packages/bpk-component-banner-alert/src/BpkBannerAlertInner.js

```diff
@@ -185,7 +185,7 @@ const BpkBannerAlertInner = (props: Props) => {
       {...rest}
     >
       <section className={sectionClassNames.join(' ')}>
-        <header
+        <div
```

Comment:
`header` isn't valid in this use case, and the test rightly caught it. It should be used as a page or section header (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)